packages/cli/test/integration/project.api.test.ts

```diff
@@ -576,19 +667,19 @@ describe('PATCH /projects/:projectId', () => {
 			const memberAgent = testServer.authAgentFor(testUser1);
 
 			const deleteSpy = jest.spyOn(Container.get(CacheService), 'deleteMany');
-			const resp = await memberAgent.patch(`/projects/${teamProject1.id}`).send({
-				name: teamProject1.name,
+			// Add two members to teamProject1
+			const addResp = await memberAgent.post(`/projects/${teamProject1.id}/users`).send({
 				relations: [
-					{ userId: testUser1.id, role: 'project:admin' },
 					{ userId: testUser3.id, role: 'project:editor' },
 					{ userId: ownerUser.id, role: 'project:viewer' },
 				] as Array<{
 					userId: string;
 					role: ProjectRole;
 				}>,
 			});
-			expect(resp.status).toBe(200);
+			expect(addResp.status === 200 || addResp.status === 201).toBe(true);
```

Comment:
It should be deterministic which status code is returned. I guess it should be 201 correct?